docs/concepts/copilot/copilot-ai-capabilities.md

```diff
@@ -0,0 +1,168 @@
+| Data Tests and Quality Checks                      | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+
+## Text to Pipelines
+
+Get started on a new Pipeline quickly by typing your prompt into the text box and Data Copilot will generate a new Pipeline or modify an existing one.
+
+### Start a new Pipeline
+
+![Start a Pipeline](img/copilot_text_to_Pipeline.png)
+
+The following example uses Data Copilot to help start a Pipeline:
+
+1. Type a prompt with English text, such as `Which customers shipped the largest orders this year?`
+2. Data Copilot uses metadata from the accessible Datasets, Seeds, Pipelines, and Models, to create a Knowledge Graph.
+3. Data Copilot uses OpenAI's language model to create the Pipeline based on the text prompt, with the Knowledge Graph as the context. This Pipeline is accessible in the visual editor as well as the code editor.
+4. If you'd like, review the suggested changes before you decide to keep or reject the suggested Pipeline. Then interactively execute it to see the results.
+5. View Data Copilot's suggested changes in the visual editor.
+
+### Modify an existing Pipeline
+
+You can also call Data Copilot to modify an existing Model. Type a new text prompt, and Data Copilot will suggest a new sequence of data transformations.
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/a2x6kkzy8y?videoFoam=true" title="Modify an existing Pipeline" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+## Next-transformation suggestions
+
+Data Copilot can suggest the next transformation in a series or the next expression within a Gem.
+
+### Suggest Gems
+
+Data Copilot can suggest the next transformation in a series as long as there is an open Tail node.
+
+![Suggest Gems](img/copilot_next_suggestion.png)
+
+See the following Join suggestion example:
+
+1. Select a Dataset of interest.
+2. Data Copilot suggests Datasets which are frequently used with the selected Dataset.
+3. Data Copilot then suggests a next transformation, in this case, a Join Gem.
+
+### Suggest Expressions
+
+At the more granular level, Data Copilot can suggest expressions within Gems.
+
+![Suggest expressions](img/copilot_next_suggestion_expression.png)
+
+Within our [advanced Expression Builder](/Spark/expression-builder) you can:
+
+1. Type an English text prompt.
+2. Data Copilot generates a code expression for a particular column.
+3. Click to accept the code expression or try again with a different prompt.
```

Comment:
There isn't an "Accept" button here; are we clicking on the suggestion to accept, and the suggestion then shows in the Expression window?

Reply:
You're right. Will update.